platform/podjs/README.md

```diff
@@ -20,21 +20,48 @@ feature. While we could support the creation of a container and a
 mechanism for loading features in `pod.js` without too much trouble,
 it is currently not implemented.
 
-The `polyOut` and `polyNav` interfaces are not implemented at the
-moment. The former because we don't use it yet, the latter because it
-is currently not part of the `Pod` interface, which we aim to rectify
-soon.
-
 ## Building
 
-Simply run `build.js` in the repository root.
+Simply run `./build.js` in the repository root.
+
+## How to set up a feature to build with podjs
+
+Follow these instructions to do so:
+
+1.  You need to point to `podjs` from the `polypoly-eu` repository and
+    add it as a "development dependency" on your project by running:
+
+    `$npm i <path-that-points-to-platform/podjs --D`
```

Comment:
nits: why the `<`? Shouldn't it be closed by `>`? Also, why *platform*? Shouldn't it be repository? Finally, `-D` uses a single dash.
Also, as I said before, indenting already assumes it's code. If you use indenting, backticks are not necessary.